### DISCUSSION TITLE
Error fheroes for mac os

### DISCUSSION THREAD

**Discussion by Amanda696969** [ihhub/fheroes2#7361] [Q&A]
### Your question

Hi, I know this issue has been addressed on these forums in the past but I can't find the solution I read the readme.txt

As an alternative to the previous step, you can manually copy the subdirectories 'ANIM',
'DATA', 'MAPS' and 'MUSIC' (some of them may be missing depending on the version of the
original game) from the original game directory to the fheroes2 installation directory.

I am not the most tech savvy person and don't know how to move forward with the next step if anybody can help it would be greatly appreciated thank you. 

<img width="798" alt="Screenshot 2023-06-27 at 10 21 12 AM" src="https://github.com/ihhub/fheroes2/assets/137913816/0053f9ad-fe8f-40fc-bd68-982b520cad70">

**Comment by zenseii** [comment]
Hi, @Amanda696969.

This step tells you to locate the installation folder of the original Heroes 2 and within this find these folders and copy them to your fheroes2 installation folder.

If you do not yet own Heroes 2 then it is available to buy here (it frequently goes for 75 % off):
https://www.gog.com/en/game/heroes_of_might_and_magic_2_gold_edition

Alternatively you can use the Heroes 2 demo by running the script called `download_demo_version.sh`.

**Comment by zenseii** [comment]
I'm converting this issue to a discussion.

**Comment by zenseii** [comment]
Please reply back if you have further questions, @Amanda696969.
